src/test/java/core/basesyntax/service/readandwrite/ReportWriterImplTest.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.readandwrite;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportWriterImplTest {
+    private static final String PATH_TO_FILE = "src/test/resources/reportTest.csv";
+    private static final String WRONG_PATH_TO_FILE = "src/test/resoces/reportTest.csv";
+    private static final String CONTENT = "fruit,quantity" + System.lineSeparator()
+            + "banana,152" + System.lineSeparator()
+            + "apple,90" + System.lineSeparator();
+    private static ReportWriter reportWriter;
+
+    @BeforeClass
+    public static void beforeClass() {
+        reportWriter = new ReportWriterImpl();
+    }
+
+    @Test
+    public void writeReport_Ok() {
```

Comment:
can we not compare booleans but read file after writing in it and compare the content?

Reply:
done